awscli/customizations/s3/subcommands.py

```diff
@@ -369,6 +369,15 @@
                         'output is suppressed.')}
 
 
+NO_PROGRESS = {'name': 'no-progress',
+               'action': 'store_false',
+               'dest': 'progress',
+               'help_text': (
+                   'File transfer progress is not displayed. This flag '
+                   'is only applied when the quiet and only-show-errors '
```

Comment:
Does the second sentence even matter? Based on the `--quiet` and `--only-show-errors` documentation, it will not show progress anyways. I would prefer to remove it in case we do not cause any confusion in the future if we add a new output parameter and forget to update this documentation.

Reply:
The second sentence was added in response to some of James' feedback.

Reply:
Oh its because it is conflicting. Yeah I'm fine with keeping it as is.